share/spack/gitlab/cloud_pipelines/configs/packages.yaml

```diff
@@ -0,0 +1,4 @@
+spack:
```

Comment:
Get rid of this line and unindent by 2 spaces. This is `packages.yaml`, not `spack.yaml`.

Reply:
whoops of course. thanks!